RHIDP-6504: Modularize modules/dynamic-plugins/proc-topology-configure (#1065)
redhat-developer/red-hat-developers-documentation-rhdh: modules/dynamic-plugins/proc-viewing-virtual-machines.adoc

[id="proc-viewing-virtual-machines_{context}"]
= Viewing virtual machines

> To view virtual machines, the OpenShift Virtualization operator must be installed and configured on a Kubernetes cluster.
> You must also grant read access to the `VirtualMachines` resource in the `ClusterRole`:
> 
> [source,yaml]
> ----
>  ...
>   apiVersion: rbac.authorization.k8s.io/v1
>   kind: ClusterRole
>   metadata:
>     name: backstage-read-only
>   rules:
>     ...
>     - apiGroups:
>         - kubevirt.io
>       resources:
>         - virtualmachines
>         - virtualmachineinstances
>       verbs:
>         - get
>         - list
> ----
> 
> To view the virtual machine nodes on the topology plugin, you must add the following code to the `kubernetes.customResources` property in the `{my-app-config-file}` file:
> 
> [source,yaml]

Reviewer: ```suggestion
.Prerequisites
. The OpenShift Virtualization operator is installed and configured on a Kubernetes cluster.

.Procedure
. Grant read access to the `VirtualMachines` resource in the `ClusterRole`:
+
[source,yaml]
----
 ...
  apiVersion: rbac.authorization.k8s.io/v1
  kind: ClusterRole
  metadata:
    name: backstage-read-only
  rules:
    ...
    - apiGroups:
        - kubevirt.io
      resources:
        - virtualmachines
        - virtualmachineinstances
      verbs:
        - get
        - list
----

. To view the virtual machine nodes on the topology plugin, add the following code to the `kubernetes.customResources` property in the `{my-app-config-file}` file:
+
[source,yaml]
```
